jenkins-agents/jenkins-agent-mvn/Dockerfile

```diff
@@ -1,5 +1,46 @@
 FROM quay.io/openshift/origin-jenkins-agent-maven:4.9
-
+USER root
 ADD settings.xml $HOME/.m2/settings.xml
+## borrowed the below code from https://github.com/browserless/chrome/blob/master/Dockerfile ##
+ARG USE_CHROME_STABLE
+
+# Application parameters and variables
+ENV APP_DIR=/usr/src/app
+ENV PLAYWRIGHT_BROWSERS_PATH=${APP_DIR}
+ENV CONNECTION_TIMEOUT=60000
+ENV CHROME_PATH=/usr/bin/google-chrome
+ENV HOST=0.0.0.0
+ENV IS_DOCKER=true
+ENV LANG="C.UTF-8"
+ENV NODE_ENV=production
+ENV PORT=3000
+ENV USE_CHROME_STABLE=${USE_CHROME_STABLE}
+
+
+RUN mkdir -p $APP_DIR &&\
+    chown -R root:root $APP_DIR
+
+WORKDIR $APP_DIR
+
+# Install app dependencies
+COPY . .
```

Comment:
I think we don't need this command as we are not copying any files here, can you remove it and test

Reply:
I have updated it and removed unused variables